resources/public/api/conf/2.0/application.yaml

```diff
@@ -926,7 +930,7 @@ paths:
             example: 2023-06-15T09:00:00Z
       responses:
         '200':
-          description: A successful get all arrivals response
+          description: A successful get includes the total count of the number of movement arrivals and all arrivals response

```

Comment:
This wording seems problematic because it is not customary to have the description of a GET response reflect any or all query string parameters for that endpoint/API operation.

Suggestion 1 (keep original wording):
A successful get all arrivals response

Suggestion 2:
A successful get all arrivals response, which may include the total count of the number of movement arrivals

Reply:
Yep - I'll keep original wording.